### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by Woody1024** [gabrieleilertsen/hdrcnn#34]
Every time I try to run the program I geht this error:

Traceback (most recent call last):
  File "hdrcnn_predict.py", line 65, in <module>
    FLAGS = tf.flags.FLAGS
AttributeError: module 'tensorflow' has no attribute 'flags'

**Comment by codeman008** [comment]
I have the same problem. Have you solved it?

**Comment by Woody1024** [comment]
﻿
Yes, I did:
The problem is the version of the python libraries I emailed the developer and they tested it with:
tensorflow 1.8 and 1.11, tensorlayer 1.4.3, OpenEXR 1.3.0.

However the problem seems to be tensorflow 2.0 anything below that should work.

Best regards,
Felix Holzapfel


Am 24.10.2019 um 10:47 schrieb codeman008 <notifications@github.com>:

﻿

I have the same problem. Have you solved it?
